### DISCUSSION TITLE
Move movies into folder ?

### DISCUSSION THREAD

**Discussion by keichitokuna** [jellyfin/jellyfin#8966] [Q&A]
Hello,

I am a noob with Jellyfin and after importing / linked all my movies to IMDb, i have quite large folder with more than 3 000 files (movies, poster, nfo....).

Is there any easy way to put the each movie and its metadate inside a folder, named like the movie ? And withour losing the ImDb sorting i manually made ?

Thanks

K.

**Comment by xyxxyxxy** [comment]
The "Auto Organize" plugin could help you here, but I am not familiar with it.

I would recommend taking a look at [Radarr](https://radarr.video/). It is able to scan your files and link them to the IMDb entry. A naming scheme can be applied and if the naming scheme includes the IMDb ID, like described [here](https://trash-guides.info/Radarr/Radarr-recommended-naming-scheme/), Jellyfin should be able to link all movies correctly after a library rescan.

**Reply by nielsvanvelzen** [reply]
The AutoOrganize plugin has been deprecated due to maintainability issues and many bugs. We don't recommend using it and it will stop working eventually as it's not updated for newer server versions anymore.
